api-admin/src/main/java/bio/terra/pearl/api/admin/service/export/ExportIntegrationExtService.java

```diff
@@ -66,6 +66,18 @@ public ExportIntegration create(
     return exportIntegrationService.create(exportIntegration);
   }
 
+  @EnforcePortalStudyEnvPermission(permission = "export_integration")
+  public void delete(PortalStudyEnvAuthContext authContext, UUID id) {
+    ExportIntegration integration =
```

Comment:
I should have written this sooner, but let's pull this into a `private ExportIntegration authToStudyEnv(StudyEnvironment, UUID integrationId)` method that's shared across the other methods in the service.  You'll need to use findWithOptions in the helper, but that's fine since there's nothign performance critical here